src/irr/asset/CBAWMeshFileLoader.h

```diff
@@ -216,7 +216,7 @@ class CBAWMeshFileLoader : public asset::IAssetLoader
     // Compatibility functions:
 
     template<uint64_t IntoVersion>
-    std::enable_if_t<IntoVersion, bool> formatConversionProlog(
+    std::enable_if_t<IntoVersion/*cannot convert into version 0*/, bool> formatConversionProlog(
```

Comment:
isn't `std::enable_if_t`c++17?

Reply:
Nah, all this `_v` and `_t` from `<type_traits>` is c++14. And i'm always checking the standard version of a feature if i'm not sure, so don't worry :)